### DISCUSSION TITLE
RAG document read error

### DISCUSSION THREAD

**Discussion by flyfox666** [open-webui/open-webui#3847] [General]
# Bug Report

## Description

After upgrading to 0.3.7 and 0.3.8, there is a strange problem with the Q&A of RAG documents.

The first conversation after uploading a document reads the document and can be answered correctly, but a subsequent question cannot be linked to the document.

Downgrading from a 0.3.8 document to 0.3.7 doesn't work either, while the log display issue in the current 0.3.8 is not yet fixed in the stable release.

Is this a problem caused by ChromaDB?

log as follow:
INFO:chromadb.api.segment:Collection user-memory-e97f841b-4429-4759-8fc9-39d9b0138df0 is not created.

![Screenshot 2024-07-13 151243](https://github.com/user-attachments/assets/6b913c7b-1d2d-4eb7-ad17-bc8b17f7c56e)

enviroment
docker based
win11

**Comment by mr-gepardas** [comment]
I am facing the same issue after upgrading, only the first message uses the uploaded file, after which all consecutive messages ignore it, like it does not exist
